### ISSUE TITLE
Jira: To prevent shut out from api, access has been blocked by Super Productivity

### ISSUE THREAD

**Issue by natea** [johannesjo/super-productivity#661]
### Your Environment

* Version used: 6.0.1 (from OSX App Store)
* Operating System and version: Mac OSX Catalina
* Desktop Environment: 
* Browser Name and version: Chrome

### Expected Behavior

The sync works without any issues.

### Current Behavior

Super Productivity is saying that access has been blocked to Jira. Why would this be happening, and what can I do to fix it? I clicked the "Unblock" button but I didn't get any kind of response as to what that actually did and if it worked.

![Screen Shot 2020-11-10 at 9 30 14 AM](https://user-images.githubusercontent.com/118984/98687196-764f1600-2337-11eb-8b90-677cea12ce82.png)

### Steps to Reproduce (for bugs)

1.
2.
3.
4.

**Comment by johannesjo** [comment]
Hey there! This is supposed to happen if there were a couple of unsuccessful attempts to request something from Jira. I assume you did enable the Jira integration for some projects? As the message states, you should check this configuration or disable it, if you don't need it.

**Comment by alexanderadam** [comment]
I have just a single project and am getting this message.
But I tested the credentials before and always got `unauthorized` back (although the credentials are definitely working in the browser) and maybe it's because of the tests before.
2FA is _not_ activated in case that's relevant.

I also tried the cookie method, but super productivity complained about the cookie value being malformed and showed an unreadable (because of it's length) error.

**Comment by johannesjo** [comment]
@alexanderadam could you maybe provide the output in your console when this happens (Ctrl+Shift+i)? This is either due to some misconfiguration or because your Jira server doesn't allow for the API to be used.

**Comment by marcitqualab** [comment]
Hello, 
I had the same issue and I have recreated the access token, then after some time the integration worked again.

**Comment by marcitqualab** [comment]
I am using Ubuntu Linux.

I got the integration working only if I unchecked the JIRA setting boxes:

- Wonky Cookie Fallback Authentication (desktop app only)
- Use Personal Access Token instead of password

In the Token/Password field I use the Access Token that I create for my Jira user in the Atlassian account security settings. I am not sure what is the Personal Access Token used for, since for me the integration is working without this checkbox. 

If I check the checkboxes above I get the error "To prevent shut out from api, access has been blocked by Super Productivity. ..." and adding the Wonky cookie in any way or format did not work for me. 

Is there any extra settings that I need to do in my JIRA account or project in order to use the Wonky cookie method of authentication?

Many thanks.

**Comment by johannesjo** [comment]
> Is there any extra settings that I need to do in my JIRA account or project in order to use the Wonky cookie method of authentication?

Not that I am aware of. But every jira installation might be slightly different I am afraid.

**Comment by marcitqualab** [comment]
Could you help me to understand better the problem? 
I might be able to do some more testing to pin-point the problem.

Is this error coming from SuperProductivity which stops sending authentication requests to JIRA after getting many failed attempts?
What is the interval of retrying the authentication requests?
Where exactly is the code to connect to JIRA to see what is the retry logic?

**Comment by johannesjo** [comment]
Each request is authenticated. So there is no login request or something like this. 

The error is very likely to be just an authentication error that comes from jira.

Most of the relevant code can be found here:
https://github.com/johannesjo/super-productivity/blob/master/electron/jira.ts
https://github.com/johannesjo/super-productivity/tree/master/src/app/features/issue/providers/jira

**Comment by SidBergstresser** [comment]
I was getting the same error. I had the Jira integration set up for two Super Productivity projects. One worked (I had set it up 6-7 months ago) but the second one wouldn't. I eventually unchecked the box for "User Personal Access Token instead of password" on the second project even though the first is set that way and is working. In both cases I'm using the API token from Atlassian. Not sure why the difference, but hopefully this will be helpful if anyone else has this issue.
